packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -190,8 +194,9 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
   }
 
   innerStyle = {
-    width: contentSize.width,
-    height: contentSize.height,
+    // TODO: If width and height aren't finite (aka is Infinity or NaN), don't use those values since it is invalid css
```

Comment:
Are you suggested we add more or different checkes?

Reply:
Ah, this one was more of a note of what I'm doing rather than a todo. Left it here as a marker for discussion, but will remove before merge if we get there